README.md

```diff
@@ -4,7 +4,7 @@
 [![Published on npm](https://img.shields.io/npm/v/@polymer/polymer.svg)](https://www.npmjs.com/package/@polymer/polymer)
 [![Published on webcomponents.org](https://img.shields.io/badge/webcomponents.org-published-blue.svg)](https://www.webcomponents.org/element/@polymer/polymer)
 
-> ℹ️ Note: This is the current stable version of the Polymer library.  At Google I/O 2018 we announced a new Web Component base class, [`LitElement`](https://github.com/Polymer/lit-element), as a successor to the `PolymerElement` base class in this library.
+> ℹ️ Note: This is the current stable version of the Polymer library.  At Google I/O 2018 we announced a new Web Component base class, [`LitElement`](https://github.com/lit/lit), as a successor to the `PolymerElement` base class in this library.
```

Comment:
I guess that since this is a readme, it should maybe point to another readme, but just checking that we don't want this to go to lit.dev

Reply:
Yeah I thought about that, but I think staying on GitHub makes sense since we're in a README already, like you say. Our Lit repo is pretty nice and prominently links to lit.dev